salt/runners/vault.py

```diff
@@ -52,6 +52,8 @@ def generate_token(
     try:
         config = __opts__.get("vault", {})
         verify = config.get("verify", None)
+        # Vault Enterprise requires a namespace
+        namespace = config.get("namespace", None)
```

Comment:
```suggestion
        namespace = config.get("namespace")
```
That extra `, None` is not required. The above code does exactly the same

Reply:
Neat. I'm pretty sure I added that for some of the testing, but if that's mechanically true, good to know.

Reply:
yep, it is:
```python
>>> a = {'b': 1}
>>> a.get('b')
1
>>> a.get('c')
>>> a.get('c', None)
>>>
```